sdk/desktopvirtualization/Azure.ResourceManager.DesktopVirtualization/api/Azure.ResourceManager.DesktopVirtualization.netstandard2.0.cs

```diff
@@ -777,6 +993,26 @@ public partial class HostPoolPatch : Azure.ResourceManager.Models.ResourceData
         public System.Collections.Generic.IDictionary<string, string> Tags { get { throw null; } }
         public string VmTemplate { get { throw null; } set { } }
     }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct HostpoolPublicNetworkAccess : System.IEquatable<Azure.ResourceManager.DesktopVirtualization.Models.HostpoolPublicNetworkAccess>
```

Comment:
This must be a typo, it should be Host**P**oolPublicNetworkAccess

Please introduce a renaming for this.